README.md

```diff
@@ -152,25 +152,7 @@ requests to the backend for execution and the backend uses the API to
 communicate with Triton.
 
 Every model must be associated with a backend. A model's backend is
-specified in the model's configuration using the 'backend' and
-'platform' settings. Depending on the backend one or the other of
-these properties is optional.
-
-* For TensorRT, 'backend' must be set to *tensorrt* or 'platform' must
-  be set to *tensorrt_plan*.
-
-* For PyTorch, 'backend' must be set to *pytorch* or 'platform' must
-  be set to *pytorch_libtorch*.
-
-* For ONNX, 'backend' must be set to *onnxruntime* or 'platform' must
-  be set to *onnxruntime_onnx*.
-
-* For TensorFlow, 'platform must be set to *tensorflow_graphdef* or
-  *tensorflow_savedmodel*. Optionally 'backend' can be set to
-  *tensorflow*.
-
-* For all other backends, 'backend' must be set to the name of the
-  backend and 'platform' is optional.
+specified in the model's configuration using the 'backend' setting. For using TensorRT backend, the value of this setting should be *tensorrt*. Similarly, for using PyTorch, ONNX and TensorFlow Backends, the `backend` field should be set to *pytorch*, *onnxruntime* or *tensorflow* respectively.
```

Comment:
Add the catch-all for other backends at the end, and break paragraph into several lines (will still render as a paragraph in markdown)
```suggestion
specified in the model's configuration using the 'backend' setting. 
For using TensorRT backend, the value of this setting should be *tensorrt*. 
Similarly, for using PyTorch, ONNX and TensorFlow Backends, the `backend` 
field should be set to *pytorch*, *onnxruntime* or *tensorflow* respectively. 
For all other backends, 'backend' must be set to the name of the backend.
```